core/src/main/java/com/novoda/rxpresso/RxPresso.java

```diff
@@ -21,31 +21,41 @@
     /**
      * @param repositories The different mocked repositories you want to control in your tests
      */
-    public RxPresso(Object... repositories) {
-        this.repositories = repositories;
+    public static RxPresso init(Object... repositories) {
```

Comment:
`RxPresso.from(Object... repositories)`?

The `from` implies that something is being created using the parameters, which might be more accurate that `init` which could suggest that the thing being initialised already exists.

It depends on the definition of "init" -> I prefer avoiding using this word because it already has a meaning in Java == "assign a value for the first time"